ldclient/feature_store.py

```diff
@@ -10,24 +10,24 @@ def __init__(self):
         self._initialized = False
         self._features = {}
 
-    def get(self, key):
+    def get(self, key, callback):
```

Comment:
yes- Locally I ran the integration harness against the redis and in-memory feature store